web-console/4.0-designs/storage/storage.md

```diff
@@ -29,3 +29,48 @@
 ![form for adding and creating storage](img/create-storage-form.png)
 - Selecting `Create Storage` exposes fields for Storage Class, Name, Access Mode, Size, and the label selector checkbox.
 - Only access modes that are compatible should be available to select in this form.
+
+## Set Default Storage Class
+- Users with appropriate privileges can set a Default Storage Class.
+- THERE ARE TWO WAY USER CAN SET A DEFAULT STORAGE CLASS:
+
+#### OPTION-1: Kebab menu option:
+
+1. Login to OpenShift > Admin > Storage (menu) > Storage classes (sub-menu)
+2. User can see list of ‘Storage classes' [list view] & there will be a label that indicates `default` storage class
+3. Click on the kebab menu > select `Set as Default Storage Class`
+
+![Set a Default Storage Class using kebab menu](img/stroage-opt1-start-1.png)
```

Comment:
Want to show an example of the user selecting the kebab on Storage Class 1 where this option is disabled to document this use case for developers? I know it was discussed in a previous round and think it would be a nice addition to the documentation here :)

Reply:
You are right. I have that screen in the image folder. I will add it.